fuzz/src/validate_valid_module.rs

```diff
@@ -30,12 +30,14 @@ pub fn run(u: &mut Unstructured<'_>) -> Result<()> {
         multi_value: config.multi_value_enabled,
         multi_memory: config.max_memories > 1,
         bulk_memory: config.bulk_memory_enabled,
-        reference_types: config.reference_types_enabled,
+        reference_types: config.gc_enabled || config.reference_types_enabled,
         simd: config.simd_enabled,
         relaxed_simd: config.relaxed_simd_enabled,
         memory64: config.memory64_enabled,
         threads: config.threads_enabled,
         exceptions: config.exceptions_enabled,
+        function_references: config.gc_enabled,
```

Comment:
mind leaving a `// TODO` here to replace this with `config.function_references_enabled` once that's implemented in `wasm-smith`?

Reply:
I think we actually don't want to treat function references as its own proposal for the most part. It is glued to wasm GC and you can't advance one through the spec process without the other at the exact same time. They are effectively the same proposal at this point.

Reply:
For example, binaryen deprecated its function references flags and just uses gc flags now.

Reply:
Added a TODO to figure out what we want to do with these feature flags, and leaving it at that.